src/ppx_deriving_yojson.ml

```diff
@@ -704,7 +680,7 @@ let desu_str_of_type_ext ~options ~path ({ ptyext_path = { loc } } as type_ext)
       Ppx_deriving.mangle_lid
         (`PrefixSuffix ("M", "of_yojson")) type_ext.ptyext_path.txt
     in
-    String.concat "." (Longident.flatten mod_lid)
+    String.concat "." (Longident.flatten_exn mod_lid)
```

Comment:
(Here again `Longident.name` would be nice.)

Reply:
Done in https://github.com/ocaml-ppx/ppx_deriving_yojson/pull/121/commits/5a2680829a9d1cb30821d296b3ec66cb56fc7cc1